### DISCUSSION TITLE
Is there anything to have a sortable list similar to django-admin-sortable2?

### DISCUSSION THREAD

**Discussion by jturmel** [viewflow/viewflow#455] [General]
I think we could get rid of most of our stock Django Admin in our current Viewflow app if there was such a thing.

**Comment by kmmbvnr** [comment]
Are you looking for inlines in form ordering or change order in a list view?

**Comment by jturmel** [comment]
Really both would be good to have just like django-admin-sortable2 supports, but primary urgency is support for it in a main listing/index, example:

![image](https://github.com/viewflow/viewflow/assets/37833/4008a6ea-3eac-43b3-a05b-66d7ee1a3f87)

**Comment by jturmel** [comment]
They have a nice animated GIF here showing how it works:
https://django-admin-sortable2.readthedocs.io/en/latest/#

Admin Sortable2 has been around for a long time, so maybe it could just build off the backbone of that?

**Comment by jturmel** [comment]
Is that enough info to go off of?

**Comment by kmmbvnr** [comment]
To implement this we could just have a custom js web ele,ent that wraps a table and custom mixing for a list view to accept POST requests

https://chatgpt.com/share/6ba9cbff-888d-4748-a064-b3588e9e467c
